app/assets/javascripts/angular/controllers/my_books.js.erb

```diff
@@ -58,4 +56,45 @@ app.controller('MyBooksCtrl', ['$scope', '$http', '$location', '$auth', function
       });
     }
   }
+
+  $scope.editBook = function(id) {
+    $scope.edit = angular.copy(getById($scope.items, id)) // Use copy to disable two way binding
+    $scope.editing = true
+    $scope.edit.condition *= 2
```

Comment:
This function is triggered when you click on the "Edit" button. It makes a copy from the original data so that user can edit $scope.edit without affecting $scope.item which is on display